ext/standard/http.c

```diff
@@ -360,3 +361,27 @@ PHP_FUNCTION(request_parse_body)
 	SG(request_parse_body_context).throw_exceptions = false;
 	memset(&SG(request_parse_body_context).options_cache, 0, sizeof(SG(request_parse_body_context).options_cache));
 }
+
+PHP_FUNCTION(http_get_last_response_headers)
+{
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	if (!Z_ISUNDEF(BG(last_http_headers))) {
+		RETURN_COPY_VALUE(return_value, &BG(last_http_headers));
```

Comment:
When using RETURN_COPY_VALUE, you should not pass return_value.

Reply:
Yeah that was a hastly change that I didn't test locally :/